### PR TITLE
Add Keepalived Tutorial [DE/EN]

### PR DESCRIPTION
**Pull request by BackInBash**
I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Markus<markus@omg-network.de>

### CODE REVIEW

File: tutorials/configure-cloud-ha/01.de.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/configure-cloud-ha"
+slug: "configure-cloud-ha"
```
Comment (rook59) [inline tutorials/configure-cloud-ha/01.de.md:4]: Please add the used software, keepalived, in the path and slug. There is more software for a HA setup.

**Review by rook59 — COMMENTED** [review]
Thanks for creating this tutorial. Everything looks good so far, just a small addition

File: tutorials/configure-cloud-ha/01.de.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/configure-cloud-ha"
```
Comment (rook59) [inline tutorials/configure-cloud-ha/01.de.md:3]: You also need `path: "/..../de"` here for the german version

File: tutorials/configure-cloud-ha/01.de.md (hunk 1 above)
Comment (BackInBash) [reply tutorials/configure-cloud-ha/01.de.md]: done

File: tutorials/configure-cloud-ha/01.de.md (hunk 2 above)
Comment (BackInBash) [reply tutorials/configure-cloud-ha/01.de.md]: done

**Review by rook59 — APPROVED** [review]

**Pull request merged**